CMakeLists.txt

```diff
@@ -1159,9 +1174,11 @@ set(DOXYGEN_STATUS "Unavailable")
 if(DOXYGEN_FOUND)
 	set(DOXYGEN_STATUS "Available (target \"doc\")")
 	# fake autotools variables
+	set(PACKAGE_VERSION ${PLAYER_VERSION})
```

Comment:
This has been successive broken in 2016 and 2018 `m(`

doxygen used the version of the expat package here (because of config file providing `PACKAGE_VERSION`)